.travis.linux.trusty.sh

```diff
@@ -0,0 +1,21 @@
+#! /bin/bash
+
+URL=ftp://ftp.simplesystems.org/pub/libpng/png/src/libpng16/libpng-1.6.23.tar.gz
+FILE=libpng-1.6.23.tar.gz
```

Comment:
My concern is for OS X — I don't see libpng on my system. Unless I'm looking in the wrong places, where would a system one come from there? Required package manager installations (e.g. Homebrew, MacPorts, etc) means a dependency on a big third-party system that might not always be available.